src/www/ios/ios-wkwebview.js

```diff
@@ -0,0 +1,9 @@
+var exec = require('cordova/exec');
+
+const WkWebKit = {
```

Comment:
I wouldn't use ES6 `const` here, since ES6 is only supported in iOS 10 webviews.